### ISSUE TITLE
FOV / Smooth Settings Display On Screen

### ISSUE THREAD

**Issue by cengizalcan** [3a1/Zodiak#5]
Hello, how can i show fov and smooth settings on CS 1.6 screen? When i press key, i don't know the settings how much. On console ECHO (FOV=50)  similar like this how can to do?

**Comment by 3a1** [comment]
I dont think so, probably will need to make a function to draw a circle in gdi, or make a function to draw a text in gdi to print the value.
